packages/app/src/PatientHeader.test.tsx

```diff
@@ -54,6 +54,22 @@ describe('PatientHeader', () => {
     expect(screen.getByText('456')).toBeInTheDocument();
   });
 
+  test('Handles null identifiers', async () => {
+    setup({
+      resourceType: 'Patient',
+      name: [
+        {
+          given: ['Alice'],
+          family: 'Smith',
+        },
+      ],
+      identifier: [null, { system: 'def', value: '456' }] as unknown as Identifier[],
```

Comment:
Do we need to test for nulls inside the dictionary? E.G.
`{system: 'def', value: null}`